lib/gooddata/models/domain.rb

```diff
@@ -191,7 +191,12 @@ def users(domain, id = :all, opts = {})
             page_limit = opts[:page_limit] || 1000
             offset = opts[:offset] || 0
             loop do
-              tmp = client(opts).get("#{domain.uri}/users", params: { offset: offset, limit: page_limit })
+              begin
+                tmp = client(opts).get("#{domain.uri}/users", params: { offset: offset, limit: page_limit })
+              rescue RestClient::BadRequest => e
+                raise e
+              end
+
```

Comment:
Is there any reason to have it here if you just raise it again without doing anything else?